client/components/domains/use-your-domain-step/index.jsx

```diff
@@ -259,7 +259,6 @@ class UseYourDomainStep extends React.Component {
 			),
 			translate( 'Manage your domain and site from your WordPress.com dashboard' ),
 			translate( 'Extends registration by one year' ),
-			translate( 'Can take several days to complete the transfer' ),
```

Comment:
Why are we removing this? It's still true, right? And it could be a factor in whether someone selects mapping or a transfer as the best solution for them.

Reply:
Sorry, I missed that this change was agreed on in p7DVsv-4Rh-p2, @aidvu ...thanks for pointing that out. :-)